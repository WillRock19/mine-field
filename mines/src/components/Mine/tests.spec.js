import 'jsdom-global/register';
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Mine from './Mine';

describe('Mine', () => {
  it('tests should be runned', () => {
    renderer.create(<Mine />);
  });

  it('should have the expected structure and style', () => {
    let snapshotTree = renderer.create(<Mine />).toJSON();
    expect(snapshotTree).toMatchSnapshot();
  });
});
