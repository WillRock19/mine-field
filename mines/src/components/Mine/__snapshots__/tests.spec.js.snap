// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mine should have the expected structure and style 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "black",
        "borderRadius": 10,
        "height": 14,
        "width": 14,
      }
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "black",
          "borderRadius": 3,
          "height": 3,
          "position": "absolute",
          "width": 20,
        },
        Object {
          "transform": Array [
            Object {
              "rotate": "45deg",
            },
          ],
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "black",
          "borderRadius": 3,
          "height": 3,
          "position": "absolute",
          "width": 20,
        },
        Object {
          "transform": Array [
            Object {
              "rotate": "90deg",
            },
          ],
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "backgroundColor": "black",
          "borderRadius": 3,
          "height": 3,
          "position": "absolute",
          "width": 20,
        },
        Object {
          "transform": Array [
            Object {
              "rotate": "135deg",
            },
          ],
        },
      ]
    }
  />
</View>
`;
